ansible/playbook.yml

```diff
@@ -0,0 +1,63 @@
+- name: Install To-Do App on new web server
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: trello_key
+      prompt: Please enter your Trello API key
+    - name: trello_token
+      prompt: Please enter your Trello app token
+    - name: trello_board_id
+      prompt: Please enter your Trello board ID
+  tasks:
+    - name: Create non-privileged todo-app user
+      user:
+        name: todo-app
+        password: $6$/kzBrrz3VjC9WutE$G7at1stilzmz2TYYqGZ0D99cQIzvkMGJJFZrOT/slqu21NoCzmEWHcR44asECjuIarZ4MQTEsizcrH50dqaAM0
```

Comment:
Looking at the [ansible user](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html) task, it looks like the password field is optional. Is there a reason for setting it here?

Reply:
Very good point, I used this to test some configuration and forgot to remove it!